lms/static/sass/discussion/views/_create-edit-post.scss

```diff
@@ -11,6 +11,11 @@
   padding: $baseline;
   max-width: 1180px;
 
+  .post-errors {
+    padding: 0 !important;
```

Comment:
Are the ``!important``s necessary? I think the intention is to never use them except in the shame file. Can you make them more specific?